### ISSUE TITLE
Proposal: deploy Ruby apps to Kubernetes 

### ISSUE THREAD

**Issue by drnic** [BrisRuby/meetups#143]
More and more Kubernetes is being introduced to small and large organizations. We’ll have a quick introduction to Kubernetes and a new layer that’s even more delightful: Google’s open source Knative platform.

**Comment by michael-harrison** [comment]
+1

**Comment by AshAngell** [comment]
+1

**Comment by toddoxford** [comment]
+1

**Comment by Mbuckley0** [comment]
@drnic Sounds great. Would you be able to give this talk at this months Brisruby and how long do you think it would go for?

**Comment by drnic** [comment]
Yep I can talk. You let me know how long you'd like me to talk and we'll see how far we get through https://starkandwayne.com/blog/introduction-to-knative/

**Comment by Mbuckley0** [comment]
@drnic Does 20-30 minutes sound good to you?

**Comment by drnic** [comment]
Ok

________________________________
